bot.py

```diff
@@ -26,57 +28,39 @@ async def on_ready():
 # Removes the default help command of discord.py to be able to create our custom help command.
 bot.remove_command("help")
 
-if __name__ == "__main__":
-	for extension in config.STARTUP_COGS:
-		try:
-			bot.load_extension(extension)
-			extension = extension.replace("cogs.", "")
-			print(f"Loaded extension '{extension}'")
-		except Exception as e:
-			exception = f"{type(e).__name__}: {e}"
-			extension = extension.replace("cogs.", "")
-			print(f"Failed to load extension {extension}\n{exception}")
```

Comment:
`bot.load_extension(extension)` requires `await`, so it had to be called in the `on_ready` function.
Also removed the error handling because it should always work.